conf/label/Default

```diff
@@ -0,0 +1,5 @@
+#e11d21 bug
```

Comment:
I think default to be same as GitHub is better. Ref: https://github.com/unknwon/sgtest/labels